docs/analyzer-rules/AZFW0006.md

```diff
@@ -0,0 +1,23 @@
+# AZFW0006: Symbol Not Found
+
+| | Value |
+|-|-|
+| **Rule ID** |AZFW0006|
+| **Category** |[FunctionMetadataGeneration]|
+| **Severity** |Error|
```

Comment:
Left a [comment about this in other PR](https://github.com/Azure/azure-functions-dotnet-worker/pull/956#discussion_r977730185).